src/lib/util/cdrom.cpp

```diff
@@ -1552,6 +1559,32 @@ static const uint16_t qoffsets[ECC_Q_NUM_BYTES][ECC_Q_COMP] =
 };
 
 
+/**
+ * @brief   -------------------------------------------------
+ *            EDC_crctable - each value represents a CRC used
+ *            in calculating the EDC for MODE1 sectors. the
+ *            CRC polynomial is from ECMA 130 section 14.3
+ *
+ *             P(X) = (X^16 + x^15 + x^2 + 1) * (x^16 + x^2 + x + 1)
+ *
+ *            where least significant parity bit (x^0) is the
+ *            most significant bit of the checksum. the CRC
+ *            table can be verified using these parameters in
+ *            Rocksoft Model CRC Algorithm.
+ *
+ *             Width   : 4 bytes
+ *             Poly    : 0x8001801bL
+ *             Reverse : TRUE
+ *
+ *            the CRC in this table are generated at runtime
+ *            with edc_crctable_init(). credit to cdrtools and
+ *            Rocksoft for the code and technical details.
+ *          -------------------------------------------------.
+ */
+
+uint32_t EDC_crctable[EDC_CRCTABLE_SIZE];
```

Comment:
Control the scope of your internal stuff.

Reply:
resolved in 2e90c84bb14a52daa515fd095f7097fa01e6f5f5